fe/fe-core/src/main/java/com/starrocks/connector/ConnectorMetadata.java

```diff
@@ -62,6 +62,8 @@ default Table getTable(String dbName, String tblName) {
 
     default void createDb(CreateDbStmt stmt) throws DdlException {}
```

Comment:
this interface can be moved

Reply:
yes, I"ll remove it later, there are some tests depend on this interface, I'll adapt the test to the new interface.